service_storagefacility/app/services/storage/NodeService.scala

```diff
@@ -45,7 +44,10 @@ trait NodeService {
       mid: MuseumId,
       nodeId: StorageNodeDatabaseId,
       envReq: EnvironmentRequirement
-  )(implicit currUsr: AuthenticatedUser): Future[Option[EnvironmentRequirement]] = {
+  )(
+      implicit currUsr: AuthenticatedUser,
+      ec: ExecutionContext
```

Comment:
But didn't we remove som of them since the import did get a bit messy and since guice will provide the same context when injected ?